src/config-manager.ts

```diff
@@ -85,21 +96,50 @@ if (process.env.PN_ACT_CONFIG_STRIPE_SECRET_KEY) {
 	};
 }
 
+// Add the old config option for backwards compatibility
```

Comment:
```suggestion
// * Add the old config option for backwards compatibility
```

We use the [Better Comments](https://marketplace.visualstudio.com/items?itemName=EdwinHuiSH.better-comments-next) comment syntax